src/applications/simple-forms/21-0845/config/submit-transformer.js

```diff
@@ -0,0 +1,38 @@
+import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
+
+import { LIMITED_INFORMATION_ITEMS } from '../definitions/constants';
+
+const escapedCharacterReplacer = (_key, value) => {
+  if (typeof value === 'string') {
```

Comment:
could you continue using our shared function for this one since it looks like it's the same?

Reply:
Done.  Now shared transformer's imported as sharedTransformForSubmit and then called below.